### DISCUSSION TITLE
Adding Custom DNS Record Type

### DISCUSSION THREAD

**Discussion by rezytijo** [tiredofit/docker-traefik-cloudflare-companion#91] [Ideas]
Hello, I have added a new Environment Variable (RC_TYPE) where later users will be able to determine the type of record that will be added to Cloudflare, I feel that this update is needed so that users can freely determine whether to use the type CNAME, A or AAAA

If anyone need it, can download from [Here](https://rezy-my.sharepoint.com/:f:/g/personal/rezyreyka7890_rezy_onmicrosoft_com/Es4IFho5zv1IoKYWF9UbukwBZxi6F-8iv61ZHlfubg-MnQ?e=sA0KRV)

https://prnt.sc/ecvCHRFHnMsQ

**Comment by tiredofit** [comment]
I'd be interested in reviewing a PR to merge this into the main codebase.
